category/execution/ethereum/execute_transaction_test.cpp

```diff
@@ -56,6 +56,18 @@ TEST(TransactionProcessor, irrevocable_gas_and_refund_new_contract)
     static constexpr auto bene{
         0x5353535353535353535353535353535353535353_address};
 
+    static auto const initial_balance = 56'000'000'000'000'000;
+    static auto const actual_gas_cost = [] {
+        if constexpr (TestFixture::Trait::evm_rev() == EVMC_FRONTIER) {
+            return 21'000;
+        }
+        else {
+            return 53'000;
+        }
+    }();
+    static auto const gas_limit = actual_gas_cost + 2'000;
```

Comment:
This is just an arbitrary value to distinguish charging gas limit vs. actual gas used?

Reply:
yes indeed